Formula/purescript.rb

```diff
@@ -18,13 +18,25 @@ class Purescript < Formula
   depends_on "ghc" => :build
   depends_on "cabal-install" => :build
 
-  def install
-    # Fix "error: Couldn't match type 'Text' with 'Line'"
-    # Upstream issue "turtle 1.3 breaks build"
-    # Reported 10 Dec 2016 https://github.com/purescript/purescript/issues/2472
-    inreplace "purescript.cabal", "turtle -any", "turtle < 1.3"
+  # Fix "Couldn't match type '[Char]' with 'Text'"
+  # Upstream issue from 2 Jan 2017 https://github.com/purescript/purescript/issues/2528
+  resource "purescript-cabal-hackage" do
+    url "https://hackage.haskell.org/package/purescript-0.10.4/purescript.cabal"
```

Comment:
I suggest to use the url https://hackage.haskell.org/package/purescript-0.10.4/revision/1.cabal as that's the one which is supposed to be persistent (i.e. its checksum won't change).